fission-cli/library/Fission/CLI/Handler/App/Publish.hs

```diff
@@ -145,37 +149,56 @@ handleTreeChanges ::
   -> JWT.Proof
   -> URL
   -> Bool
-  -> MVar UTCTime
-  -> MVar Text
+  -> TVar UTCTime
+  -> TVar Text
   -> WatchManager
   -> FilePath -- ^ Build dir
   -> IO StopListening
 handleTreeChanges runner userProof appURL copyFilesFlag timeCache hashCache watchMgr absDir =
-  FS.watchTree watchMgr absDir (\_ -> True) \_ -> runner do
-    now     <- getCurrentTime
-    oldTime <- readMVar timeCache
-
-    unless (diffUTCTime now oldTime < Time.doherty) do
```

Comment:
This started to bother the Rubyist in me. `unless` leads to a lot of double negatives.

Reply:
👍